src/appdev/ApplicationRouter.js

```diff
@@ -85,6 +86,10 @@ class ApplicationRouter<T> {
     throw new Error('You must implement getPath() with a valid path!');
   }
 
+  getMiddleware(): ?Array<Function> {
```

Comment:
Can this function follow the same format as Pollo's? https://github.com/cuappdev/pollo-backend/blob/master/src/utils/AppDevRouter.js